README.md

```diff
@@ -32,26 +32,40 @@ The optional parameters are:
 13. ***STARTUP_COMMAND***. This defines the startup command to run when the VM starts up. Ex: python /usr/cp/train.py
 14. ***ORCHESTRATOR_TYPE***. This defines the Orchestrator type to be set up on the VMs. The currently supported orchestrator types are 
     -  __ray__: A Ray cluster is created using the MIG instances.
-    -  __gke__: A private GKE cluster is created with private nodepool following the recommendations from the GKE team.
+    -  __gke__: A private GKE cluster is created with private node pool following the recommendations from the GKE team.
 15. ***GKE_NODE_POOL_COUNT***: The number of homogeneous node pools for GKE cluster. Only applicable when `ORCHESTRATOR_TYPE` is `gke`.
 16. ***GKE_MIN_NODE_COUNT***: The minimum node count of the node pools for GKE cluster. Only applicable when `ORCHESTRATOR_TYPE` is `gke`.
 17. ***GKE_MAX_NODE_COUNT***:The maximum node count of the node pools for GKE cluster. Only applicable when `ORCHESTRATOR_TYPE` is `gke`.
-18. ***GCS_MOUNT_LIST***. This defines the list of GCS buckets to mount. The format is `<bucket1>:</mount/path1>,<bucket2>:</mount/path2>`. For example: GCS_MOUNT_LIST=test-gcs-bucket-name:/usr/trainfiles
-19. ***NFS_FILESHARE_LIST***. This defines the list of NFS file shares to mount. The format is `</mount/path1>:<NFS fileshare type>,</mount/path2>:<NFS fileshare type>:<NFS fileshare size in GB>`. For example: NFS_FILESHARE_LIST=/usr/nfsshare1:BASIC_SSD
+18. ***CUSTOM_NODE_POOL***: The custom node pool description for GKE. The structure of the custom node pool is list of node pool objects. The node pool object is 
+    ```
+    name                    = string
+    nodes_initial           = number
+    nodes_min               = number
+    nodes_max               = number
```

Comment:
I think what we need in here is `node_initial` and `nodes_preferred`.
min and max terminology immediately makes me think about auto-scaling, instead of this.